config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
-  devise_for :users
   root to: 'pages#home'
+  devise_for :users do
```

Comment:
Just looking at our routes from yesterday, I *think* bookings needs to be nested in cars as opposed to users?

Reply:
good spot - yes!